src/test/java/seedu/address/logic/parser/DeleteCommandParserTest.java

```diff
@@ -25,8 +28,30 @@ public void parse_validArgs_returnsDeleteCommand() {
         assertParseSuccess(parser, " i/1", new DeleteCommand(ID_FIRST_PERSON));
     }
 
+    @Test
+    public void parse_invalidId_throwsParseException() {
+        // zero id
+        assertParseFailure(parser, " i/0", Id.MESSAGE_CONSTRAINTS);
+
+        // negative id
+        assertParseFailure(parser, " i/-5", Id.MESSAGE_CONSTRAINTS);
+
+        // non numerical id
+        assertParseFailure(parser, " i/one", Id.MESSAGE_CONSTRAINTS);
+
+        // empty id
+        assertParseFailure(parser, " i/", Id.MESSAGE_CONSTRAINTS);
+    }
+
+    @Test
+    public void parse_repeatedId_throwsParseException() {
```

Comment:
perhaps instead of repeatedId, we could duplicatePrefix for naming?

Reply:
Sounds more accurate! 👍